gcp/modules/gpii-flowmanager/resources/flowmanager-couchdb-network-policy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: flowmanager-couchdb-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: couchdb
```

Comment:
Since this is a rule "about" couchdb, I think this file should be under `couchdb/resources` (and renamed `couchdb-flowmanager-network-policy). What do you think?

Reply:
I would continue to argue that it belongs with `flowmanager`. 

In the general form, if a document defines the Policy between a `service A` (eg. flowmanager) and its dependency `service  B` (eg. couchdb), then placing the Policy with `service A` will provide for stronger decoupling between the two services. If we decide that `service A` is no longer required, then all we have to do is remove `service A` from the deployment logic. 

If we place the policy with `service B`, then we'll have to remember to modify the policy with `service B` upon deprecation of `service A`.

Reply:
Imho this rule should sit with flowmanager application.

There will need to be 2 rules if we want to do both ingress and egress - one allowing egress for the app itself, one allowing ingress to the target resource from the app.

Why with the app? If we add/change/remove app, I would prefer to modify it only in the app module and not have to modify CouchDB module every time.

Reply:
Thanks for explaining the rationale @seg10 @stepanstipl. I'll close this.